Jenkins_jobs/WorkspaceInfo.groovy

```diff
@@ -141,6 +101,10 @@ node {
 
         cleanWs()
     }
+
+    stage('Gather_Space_Info') {
+        parallel clones
```

Comment:
Running the "clones" while still occupying the original node is a perfect recipe for deadlock. If we need the node block thats fine, but release the node before running the parallel.

Reply:
ahh yes, that's true. I'll move clones out of the node